package.json

```diff
@@ -12,38 +12,32 @@
   "files": [
     "README.md",
     "lib/*",
-    "lib/dynamodb/**",
-    "lib/services/**",
     "package.json"
   ],
   "scripts": {
-    "build": "yarn run build:babel && yarn run build:flow-copy-source && yarn run build:license-checker",
-    "build:babel": "babel src --out-dir lib --ignore test.js",
-    "build:flow-copy-source": "flow-copy-source --ignore test.js src lib",
+    "build": "yarn run build:license-checker && tsc",
+    "build:flowtypes": "find lib -type f -name '*.d.ts' -exec sh -c 'yarn flowgen --add-flow-header $1 -o ${1%.*.*}.js.flow' _ '{}' \\;",
     "build:license-checker": "license-checker --csv --out var/licenses.csv",
     "clean": "shx rm -rf lib",
+    "flow": "echo this is not a flow project",
```

Comment:
@jeffsays - there's a workflow script w/ a `yarn flow start` so I left this behind

Reply:
i like it